packages/forge/blocks/segment/actions/identify.ts

```diff
@@ -0,0 +1,71 @@
+import { createAction, option } from '@typebot.io/forge'
+import { Analytics } from '@segment/analytics-node'
+import { auth } from '../auth'
+
+export const identify = createAction({
+  auth,
+  name: 'Identify User',
+  options: option.object({
+    userId: option.string.layout({
+      label: 'User ID',
+      isRequired: true,
+      helperText: 'Identifier of the user to track the event for.',
+    }),
+    email: option.string.layout({
+      label: 'Email',
+      isRequired: true,
```

Comment:
Is it required?

Reply:
Not strictly, could be optional. Practically speaking with Segment, it is almost always used. But I can make this optional.

Reply:
Email is optional now

Reply:
Thanks